spacy_llm/tasks/spancat/registry.py

```diff
@@ -119,4 +126,72 @@ def make_spancat_task_v2(
         single_match=single_match,
         spans_key=spans_key,
         scorer=scorer or score,
+        description=None,
+        check_label_consistency=check_labels,
+    )
+
+
+@registry.llm_tasks("spacy.SpanCat.v3")
+def make_spancat_task_v3(
+    parse_responses: Optional[TaskResponseParser[SpanCatTask]] = None,
+    prompt_example_type: Optional[Type[FewshotExample]] = None,
+    labels: Union[List[str], str] = [],
+    template: str = DEFAULT_SPANCAT_TEMPLATE_V3,
+    description: Optional[str] = None,
+    label_definitions: Optional[Dict[str, str]] = None,
+    examples: ExamplesConfigType = None,
+    normalizer: Optional[Callable[[str], str]] = None,
+    alignment_mode: Literal["strict", "contract", "expand"] = "contract",
+    case_sensitive_matching: bool = False,
+    spans_key: str = "sc",
+    scorer: Optional[Scorer] = None,
+    check_label_consistency: Optional[SpanTaskLabelCheck[SpanCatTask]] = None,
+):
+    """SpanCat.v3 task factory for SpanCat with chain-of-thought prompting.
+
+    parse_responses (Optional[TaskResponseParser[SpanCatTask]]): Callable for parsing LLM responses for this
+        task.
+    prompt_example_type (Optional[Type[FewshotExample]]): Type to use for fewshot examples.
+    labels (Union[str, List[str]]): List of labels to pass to the template,
+        either an actual list or a comma-separated string.
+        Leave empty to populate it at initialization time (only if examples are provided).
+    template (str): Prompt template passed to the model.
+    description (str): todo
```

Comment:
TODO

Reply:
Done in https://github.com/explosion/spacy-llm/pull/180/commits/3a31530bfd257cd77028a964e9b1e421676c0abd.